src/accelerate/commands/config/config_args.py

```diff
@@ -109,8 +109,6 @@ def from_json_file(cls, json_file=None):
             config_dict["use_cpu"] = False
         if "debug" not in config_dict:
             config_dict["debug"] = False
-        if "enable_cpu_affinity" not in config_dict:
-            config_dict["enable_cpu_affinity"] = False
```

Comment:
We certainly can't have this, this removes CPU affinity entirely :) Can you please state what the error is in sagemaker? Maybe we need to add it to the sagemaker config

Reply:
Sagemaker doesnt support this arg